content/react-lsn3/studio/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Studio"
+date: 2023-04-12T16:25:46-05:00
+draft: false
+weight: 3
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Let's add some additional functionality to our recipe app! To get started, check out the starter code for the studio in the `part3` directory on the `main` branch of `react-exercises-and-studios`. The starter code includes all the components you will need and in `App.js`, all the components are laid out for you. However, if you run the code, you will find lots of warnings and errors. Time to check out the different components and make use of what you have learned about state!
```

Comment:
Suggestion to add links to the GitHub Repo.

Reply:
Done!